### DISCUSSION TITLE
Configure Vimwiki Plugin

### DISCUSSION THREAD

**Discussion by rotlaus** [LazyVim/LazyVim#258] [Q&A] [answered]
Hi, i'm trying to get vimwiki working in Lazyvim, but it doesn't seem to read my config. I've added a lua/plugins/wiki.lua:
```lua
return {
  {
    "vimwiki/vimwiki",
    config = function()
      vim.g.vimwiki_list = {
        { path = '~/st/pm/notes', syntax = 'markdown', ext = '.md', links_space_char = '-' },
        { path = '~/st/pm/work_notes', syntax = 'markdown', ext = '.md', links_space_char = '-' },
      }
      vim.g.vimwiki_ext2syntax = { ['.md'] = 'markdown', ['.markdown'] = 'markdown', ['.mdown'] = 'markdown' }
      vim.g.vimwiki_use_mouse = 1
      vim.g.vimwiki_markdown_link_ext = 1
    end,
  },
}
```
It looks like the variables get set properly, an `echo g:vimwiki_list` shows the correct settings. But starting Vimwiki with `VimwikiIndex` tries to create the wiki in the default location ~/vimwiki. Vimwiki seems to ignore my settings.
What am i doing wrong?

**Comment by folke** [comment, marked as answer]
You should set those variables in `init`, so just change `config` to `init`

**Reply by rotlaus** [reply]
That worked. Thank you so much.

**Comment by smokeyhallow** [comment]
@rotlaus did vimwiki work for you, such that the enter to make links or navigate to them worked?

**Reply by rotlaus** [reply]
Yes, works just fine. The configuration shown above is all I have, nothing extra needed.
